ipa_3d_fov_visualization/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(ipa_3d_fov_visualization)
+
+
+find_package(catkin REQUIRED COMPONENTS
+            cv_bridge
+            image_geometry
+            roscpp
+            sensor_msgs
+            visualization_msgs
+            )
+
+catkin_package()
+
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
+)
+
+
+add_executable(ipa_3d_fov_visualization src/ipa_3d_fov_visualization.cpp)
+
+target_link_libraries(ipa_3d_fov_visualization ${catkin_LIBRARIES})
+
```

Comment:
I'd still insist on the consistent order:
 - `add_executable`
 - `add_dependencies`
 - `target_link_libraries`